### DISCUSSION TITLE
Ignoring everything that looks like anchors (<XYZ> & </XYZ>) in templates

### DISCUSSION THREAD

**Discussion by oxtna** [JanDeDobbeleer/oh-my-posh#3483] [Troubleshoot] [answered]
A couple of days ago I updated oh-my-posh after a long time and a ligature I used for shortening my path (`</>`) stopped showing up.

After a bit of troubleshooting, I found out that anything in angle brackets gets recognised as an anchor (by the regex) and oh-my-posh tries to interpret it as a color, and after it fails to do so, it defaults to some foreground color, skipping the anchor-like text overall.

Is this the intended behavior?

Edit: Just in case, I realize that `</>` is now reserved.

Thanks

**Comment by JanDeDobbeleer** [comment, marked as answer]
@oxtna yes, </> is reserved. There's no other way unfortunately.
